sphinx/ext/autosummary/__init__.py

```diff
@@ -790,5 +790,6 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_config_value('autosummary_mock_imports',
                          lambda config: config.autodoc_mock_imports, 'env')
     app.add_config_value('autosummary_imported_members', [], False, [bool])
+    app.add_config_value('autosummary_filename_map', {}, 'html')
```

Comment:
It is better to add a new config variable alphabetically. Could you move this above?
(I noticed `autosummary_imported_members` is not. I'll fix it later)